kafka/src/main/java/io/specmesh/kafka/provision/Provisioner.java

```
@@ -79,7 +79,7 @@ public static class ProvisioningException extends RuntimeException {
         }
 
         ProvisioningException(final String msg, final Throwable cause) {
-            super(msg, cause);
+            super(msg + "|" + cause.getMessage());
```

.getMessage() this will swallow upchain exceptions